File: data/code_pairs/cpp/1713.cpp
#include <string>
#include <vector>

std::string copyFirstElement(const std::vector<std::string>& wbuf2) {
    std::string wbuf1(2, ' ');
    if (!wbuf2.empty()) {
        wbuf1 = wbuf2[0].substr(0, 2);
    }
    return wbuf1;
}

File: data/code_pairs/fortran/1713.f90
! Main code
subroutine copyFirstElement(wbuf1, wbuf2)
    character(len=2), intent(out) :: wbuf1
    character(len=2), dimension(:), intent(in) :: wbuf2
    wbuf1 = wbuf2(1)
end subroutine copyFirstElement